### PR TITLE
restructuring poms to link server module to integration-testing,

### PR DESCRIPTION
**Pull request by albertshau**
consolidating versions and common dependencies.

### CODE REVIEW

File: integration-testing/pom.xml [hunk 1]
```diff
@@ -21,39 +21,17 @@
...
-        </repository>
-        <repository>
-            <id>continuuity-release</id>
-            <url>https://repository.continuuity.com/content/repositories/releases-public/</url>
-        </repository>
-    </repositories>
+    <parent>
+        <groupId>com.continuuity</groupId>
+        <artifactId>loom-parent</artifactId>
+        <version>0.9.7-SNAPSHOT</version>
```
Comment (wolf31o2) [inline integration-testing/pom.xml:30]: 0.9.7-SNAPSHOT or 0.9.6-SNAPSHOT?

I am guessing the former is supposed to mean 0.9.7-somelargenumberhere and is supposed to be the equivalent of 0.9.7_pre?

Comment (albertshau) [inline integration-testing/pom.xml:30]: Correct, 0.9.7-SNAPSHOT means it's what will eventually become 0.9.7.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>2.9</version>
+                    <configuration>
+                        <excludePackageNames>*.internal.*</excludePackageNames>
+                        <links>
+                            <link>http://download.oracle.com/javase/6/docs/api/</link>
+                        </links>
+                        <bottom>
+                            <![CDATA[Copyright 2013 Continuuity, Inc. <a href="http://www.apache.org/licenses/LICENSE-2.0">Licensed under the Apache License, Version 2.0</a>]]>
```
Comment (wolf31o2) [inline pom.xml]: 2013-2014

Comment (albertshau) [inline pom.xml]: good catch

**Comment by wolf31o2** [comment]
Looks good. I really dislike the top-level pom.xml, since it isn't for the entire project, but that shouldn't block merge.

**Pull request merged**